pkgs/by-name/ch/chance/package.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   cargoDeps = rustPlatform.fetchCargoTarball {
-    inherit (finalAttrs) pname version src;
+    inherit (finalAttrs) src;
```

Comment:
According to the PR headmatter, this is removed in order to not cause hash mismatches.